### PR TITLE
CVSL-95: Take User roles into account to display the correct journey cards on the landing page

### PR DESCRIPTION
**Pull request by smcveigh941**
- Add user roles into the express.User object
- Util function to check for a specific role for the user
- Display cards on landing page depending on whether certain roles exist
- Removed the roles  'ROLE_CVL_OMU' and 'ROLE_GLOBAL_SEARCH' - replaced by relevant licence roles
- Started new cypress test for create a licence journey to be built on later

### CODE REVIEW

File: server/routes/home/handlers/home.test.ts [hunk 1]
```diff
@@ -18,7 +21,11 @@ describe('Route Handlers - Home', () => {
   describe('GET', () => {
     it('', async () => {
       await handler.GET(req, res)
-      expect(res.render).toHaveBeenCalledWith('pages/index')
+      expect(res.render).toHaveBeenCalledWith('pages/index', {
+        shouldShowCreateLicenceCard: true,
+        shouldShowApproveLicenceCard: true,
+        shouldShowMyCaseloadCard: true,
+      })
```
Comment (timharrison-moj) [inline server/routes/home/handlers/home.test.ts:28]: This is nice.

File: integration_tests/integration/createLicence.spec.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const IndexPage = require('../pages/index')
+
+context('Create a Licence', () => {
```
Comment (timharrison-moj) [inline integration_tests/integration/createLicence.spec.js:3]: Should there be an Axe test here to verify that the page is accessible? Or better to put these accessibility checks in a different test?

**Review by timharrison-moj — APPROVED** [review]
LGTM. One question about an Axe test, but could be a follow-on PR

File: integration_tests/integration/createLicence.spec.js (hunk 2 above)
Comment (smcveigh941) [reply integration_tests/integration/createLicence.spec.js:3]: Added Axe testing to the verifyPage function - this allows accessibility to be tested implicitly each time a page is loaded during the test

**Review by timharrison-moj — APPROVED** [review]
Sorted.

**Pull request merged**